lisa/base_tools/apt_add_repository.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import cast
+
+from lisa.executable import Tool
+
+
+class AptAddRepository(Tool):
+    @property
+    def command(self) -> str:
+        return "apt-add-repository"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def install(self) -> bool:
+        from lisa.operating_system import Ubuntu
+
+        ubuntu_os: Ubuntu = cast(Ubuntu, self.node.os)
+        package_name = "software-properties-common"
+        ubuntu_os.install_packages(package_name)
+        return self._check_exists()
+
+    def add_repository(self, repo: str) -> None:
+        # create folder when it doesn't exist
```

Comment:
remove this comments

Reply:
Removed as requested. The dangers of copy paste programming.